src/sidebar-overlay/content_script/rendering.ts

```diff
@@ -51,7 +51,7 @@ export function createShadowRootIfSupported(
 
     let shadow = null
     if (container.attachShadow) {
-        shadow = container.attachShadow({ mode: 'closed' })
+        shadow = container.attachShadow({ mode: 'open' })
```

Comment:
What's the reasoning behind this change?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Element/attachShadow#Parameters
I had to change this to access drag element inside shadow dom(`dragged-element`) and change the style `display:block` so when tried to drag page from sidebar to collections list text (Drop into Collection) can be displayed

Reply:
Cool, that makes sense. Let's put another comment here explaining this, so someone doesn't change it back later and break that functionality.